.github/workflows/ci.yml

```diff
@@ -82,37 +80,50 @@ jobs:
         phpize
         ./configure
         make
-        make test
-
-
-  #   linux-arm:
-  #     name: ARM (${{ matrix.arch }})
-  #     runs-on: ubuntu-latest
-  #     strategy:
-  #       matrix:
-  #         arch: [armv6, armv7, arm64]
-  #     steps:
-  #     - name: Build ${{ env.PACKAGE_NAME }}
-  #       run: |
-  #         python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
-  #         chmod a+x builder
-  #         ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
-
-  # windows-vc16:
-  #   runs-on: windows-latest
-  #   strategy:
-  #     matrix:
-  #       arch: [x64]
-  #   steps:
-  #   - uses: ilammy/msvc-dev-cmd@v1
-  #     with:
-  #       arch: ${{ matrix.arch }}
-  #       uwp: false
-  #       spectre: true
-  #   - name: Build ${{ env.PACKAGE_NAME }} + consumers
-  #     run: |
-  #       python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
-  #       python builder.pyz build -p ${{ env.PACKAGE_NAME }} --spec=downstream
+        ./dev-scripts/run_tests
+
+    # linux-arm:
+    #   name: ARM (${{ matrix.arch }})
+    #   runs-on: ubuntu-latest
+    #   strategy:
+    #     matrix:
+    #       arch: [armv6, armv7, arm64]
+    #   steps:
+    #   - name: Build ${{ env.PACKAGE_NAME }}
+    #     run: |
+    #       python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
+    #       chmod a+x builder
+    #       ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
+
+  windows-vc16:
+    runs-on: windows-2019
+    defaults:
+      run:
+        shell: cmd # use CMD instead of powershell to catch error from bat script
+    strategy:
+      matrix:
+        arch: [x64]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - id: setup-php-sdk
+      uses: cmb69/setup-php-sdk@v0.6
+      with:
+        version: '8.0'
+        arch: x64
+        ts: ts
+        deps: openssl
+    - uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ matrix.arch }}
+        toolset: ${{steps.setup-php-sdk.outputs.toolset}}
+    # CMD only execute one commend per run
```

Comment:
I don't know... 

I used to do 
```
- run: |
   phpize
   ./configure ***
```
It only runs phpize, so, I guess that's why

Reply:
I'm not used to seeing `./` syntax on windows. Maybe that's confusing it?

Try running `configure` or `.\configure` instead of ~./configure~

Reply:
pretty sure if you run this like `configure` or `.\configure` vs ~./configure~ it would all work

I just have to leave a "maybe not really true" comment like this in the repo, forever passing on superstitions to some poor newcomer

Reply:
https://github.com/awslabs/aws-crt-php/pull/86/commits/830ac2d84176ee15eeb2a999f75c415de46b1a1a

I believe I used `.\configure`, sorry I just typed wrong from the conversation above, already got used to `/`....

Reply:
yeah, who knows 🤷‍♀️. I give up